src/test/java/me/gnat008/perworldinventory/config/SettingsConsistencyTest.java

```diff
@@ -107,7 +109,7 @@ public void shouldHavePublicStaticFinalFields() {
     @Test
     public void shouldHaveUniquePaths() {
         Set<String> paths = new HashSet<>();
-        for (Class<?> clazz : classes) {
+        for (Class<? extends SettingsHolder> clazz : classes) {
             Field[] fields = clazz.getDeclaredFields();
             for (Field field : fields) {
                 if (Property.class.isAssignableFrom(field.getType())) {
```

Comment:
Cant add a comment to the line below where this actually happens, but I'm getting this:
http://hastebin.com/zihawixomi.vbs